### PR TITLE
Add brewservicesmenubar v2.1.0

### PR DESCRIPTION
**Pull request by stefansundin**
Hi! This is my first cask that I'm contributing, so let me know if I missed anything. Thanks!

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by stefansundin** [comment]
Weird.. I followed the instructions here: 
https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md#required-appcast-parameters

```
$ brew cask _appcast_checkpoint --calculate https://github.com/andrewn/brew-services-menubar/releases.atom
d80195ef36d527012f9ae3a1e78556de090ef950548bd0c3cb767810f3b52a58
```

**Comment by gtmgianni** [comment]
@stefansundin 

```
$ brew cask _appcast_checkpoint --calculate https://github.com/andrewn/brew-services-menubar/releases.atom
bb694b2453628becf0888b949205e020db1ad9381e5412904d0ad85383f5917c
```

Not sure why you're getting a different result.

**Comment by stefansundin** [comment]
I figured it out. I have a `~/.curlrc` file which is causing it. I'll try to submit a separate PR to fix that issue, if I can find the code that is invoking curl.

**Comment by commitay** [comment]
>  I have a ~/.curlrc file which is causing it. I'll try to submit a separate PR to fix that issue, if I can find the code that is invoking curl.

`brew doctor` already has a warning about this.

File: Casks/brewservicesmenubar.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-cask 'brew-services-menubar' do
+cask 'brewservicesmenubar' do
```
Comment (commitay) [inline Casks/brewservicesmenubar.rb]: [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md)

**Review by commitay — APPROVED** [review]

**Pull request merged**